packages/gatsby-source-drupal/src/utils.js

```diff
@@ -1,11 +1,74 @@
 const _ = require(`lodash`)
+const axios = require('axios')
 const { nodeFromData, downloadFile, isFileNode } = require(`./normalize`)
 
 const backRefsNamesLookup = new WeakMap()
 const referencedNodesLookup = new WeakMap()
 
-const handleReferences = (node, { getNode, createNodeId }) => {
+const fetchLanguageConfig = async ({ translation, baseUrl, apiBase, basicAuth, headers, params }) => {
+  if (!translation) {
+    return {
+      defaultLanguage: 'und',
+      enabledLanguages: ['und'],
+      translatableEntities: [],
+    }
+  }
+
+  const [availableLanguagesResponse, translatableEntitiesResponse] = await Promise.all([
+    axios.get(`${baseUrl}/${apiBase}/configurable_language/configurable_language`, {
+      auth: basicAuth,
+      headers,
+      params,
+    }),
+    axios.get(`${baseUrl}/${apiBase}/language_content_settings/language_content_settings?filter[language_alterable]=true`, {
+      auth: basicAuth,
+      headers,
+      params,
+    }),
+  ])
+
+  const enabledLanguages = (
+    availableLanguagesResponse
+    .data
+    .data
+    .filter(
+      language => [
+        `und`,
+        `zxx`
+      ].indexOf(language.attributes.drupal_internal__id) === -1
+    )
+    .map(language => language.attributes.drupal_internal__id)
+  )
+
+  const defaultLanguage = enabledLanguages[0]
```

Comment:
The default language is not always the first in the array. The default language of a site is stored in system.site.yml -> default_langcode
I think this value is not exposed via jsonapi. We might have to set it in the gatsby-config.js

Reply:
I'll bring this up in the #contenta slack channel tomorrow to see if anyone has ideas on how to get the default langauge via the JSON:API.

Reply:
It's my understanding that the lowest weight language will be default so perhaps the request just needs to be sorted by weight.

Reply:
I changed the request url to `/configurable_language/configurable_language?sort=weight` and it seems to work. I am testing the package now and so far its looking good. Good job! Thank you.

Reply:
I'll update this patch to include your changes.